ANSWERS.md

```diff
@@ -0,0 +1,19 @@
+1. What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.
+
+Proptypes are used to check data being passed down via props. It helps catch errors and gives you a better Idea of what the errors mean.
+
+2. Describe a life-cycle event in React?
+
+A function that gives us the ability to mount and unmount a component. This way you store it on local storage to be called back after a page refresh.
+
+3.Explain the details of a Higher Order Component?
+ 
+ A function that takes in another component and returns a new component while not modifing the input component.
```

Comment:
Nice catch!

"Note that a HOC doesn’t modify the input component, nor does it use inheritance to copy its behavior. Rather, a HOC composes the original component by wrapping it in a container component. A HOC is a pure function with zero side-effects."

https://reactjs.org/docs/higher-order-components.html